### ISSUE TITLE
[BUG] - Niet compliant volgens axereport

### ISSUE THREAD

**Issue by Pieter-Beckers** [milieuinfo/webcomponent-vl-ui-select#60]
**Omschrijf het probleem**
Volgens de pwa-helpers/axe-report is de component in dressed state niet conform aan de accessibility. Te bekijken of dit echt WCAG compliancy hier mee afcheckt en of dit een probleem is bij de component zelf.

**Hoe te reproduceren**

```
import 'axe-core/axe.min.js';

axeReport(*dressed select element*)
```
**Gewenst gedrag**
Geen violations.

**Log**
```
Error: Accessibility Violations
---
ARIA input fields have an accessible name
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute or the title attribute is empty
<div class="vl-select__list" dir="ltr" role="listbox">
---
Form elements must have labels
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty
<input type="text" class="vl-input-field
            vl-input-field-cloned" autocomplete="off" autocapitalize="off" spellcheck="false" role="textbox" aria-autocomplete="list" placeholder="Zoek item">
---
                  axeReport at /components/pwa-helpers/axe-report.js:82
```

**Omgeving (gelieve aan te vullen):**
 - Chrome web driver, via WCT framework

**Extra info**
/

**Comment by coemans** [comment]
AXE integratie is lopende, waardoor we dit issue als duplicate beschouwen en afsluiten.
